### PR TITLE
Add create methods to use POST endpoints to create analytics/player licenses

### PR DESCRIPTION
**Pull request by Svarozic**
https://github.com/bitmovin/dashboard/issues/642

### CODE REVIEW

**Comment by Svarozic** [comment]
> Is it not possible to add the return type here?

Could you make it like code comment, so I would be able to see place where ?

File: tests/analytics/licenses.test.ts [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+  assertPayload,
+  mockHttp,
+  mockPost,
+  testSetup
+} from '../assertions';
+import {getConfiguration} from '../utils';
+
+const testConfiguration = getConfiguration();
+
+describe('player', () => {
```
Comment (lukaskroepfl) [inline tests/analytics/licenses.test.ts]:
```suggestion
describe('analytics', () => {
```

File: tests/player/licenses.test.ts [hunk 2]
```diff
@@ -36,7 +42,7 @@ describe('player', () => {
 
     describe('update', () => {
       assertItCallsCorrectUrl('PUT', '/v1/player/licenses/my-license-id', () =>
-        licensesClient('my-license-id').update()
+        licensesClient('my-license-id').update({})
```
Comment (lukaskroepfl) [inline tests/player/licenses.test.ts]: does it fail now when not passing an empty object?

Comment (Svarozic) [reply tests/player/licenses.test.ts]: no it does not fail, I have just fixed typescript error , as argument of that function can not be  `undefined` but it has to be `UpdatePlayerLicense`, I think this is a left over from somebody else from before

File: bitmovin/analytics/licenses.ts [hunk 3]
```diff
@@ -21,6 +21,11 @@ export const licenses = (configuration, httpClient: HttpClient) => {
     };
   };
 
+  const create = licensePayload => {
```
Comment (Domysee) [inline bitmovin/analytics/licenses.ts]: Here basically

**Pull request merged**